apimonitor/apimonitor/settings.py

```
@@ -0,0 +1,184 @@
+}
+
+# Celery Configuration
+CELERY_BROKER_URL = env.str('REDIS_URL', 'redis://localhost:6379/0')
+CELERY_RESULT_BACKEND = env.str('REDIS_URL', 'redis://localhost:6379/0')
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json'
+CELERY_TIMEZONE = TIME_ZONE
+
+# Celery Beat Schedule
+from celery.schedules import crontab
```

Ruff (0.12.2)

Module level import not at top of file

(E402)